.abs/main.yaml

```diff
@@ -0,0 +1,5 @@
+replace-chart-version-with-git: true
+generate-metadata: true
+chart-dir: ./helm/athena
+destination: ./build
+catalog-base-url: https://giantswarm.github.io/giantswarm-playground-catalog/
```

Comment:
We are going to have to move to `giantswarm` catalog (before actually releasing?)